types/algebra.js/index.d.ts

```diff
@@ -5,58 +5,58 @@
 // TypeScript Version: 2.2
 
 declare class Term {
-  coefficients: algebra.js.Fraction[];
-  variables: Variable[];
-  coefficient(): algebra.js.Fraction;
-  toString(): string;
+    coefficients: algebra.js.Fraction[];
+    variables: Variable[];
+    coefficient(): algebra.js.Fraction;
+    toString(): string;
 }
 
 declare class Variable {
-  variable: string;
-  toString(): string;
+    variable: string;
+    toString(): string;
 }
 
 type Union = string | number | algebra.js.Fraction | Term;
 
 declare namespace algebra.js {
-  function parse(input: string): Equation | Expression;
-  function toTex(input: Fraction | Expression | Equation | object | Array<Fraction | object>): string;
+    function parse(input: string): Equation | Expression;
+    function toTex(input: Fraction | Expression | Equation | object | Array<Fraction | object>): string;
 
-  class Equation {
-    lhs: Expression;
-    rhs: Expression;
-    constructor(lhs: Expression, rhs: Expression | Fraction | number)
-    solveFor(variable: string): Fraction | Fraction[] | number[];
-    toString(): string;
-  }
+    class Equation {
+        lhs: Expression;
+        rhs: Expression;
+        constructor(lhs: Expression, rhs: Expression | Fraction | number);
+        solveFor(variable: string): Fraction | Fraction[] | number[] | Expression | undefined;
```

Comment:
this is the only change other than formatting right?

Reply:
i only added this part: "| Expression | undefined" the rest is formating